db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
```

Comment:
The condition name should only describe what the condition is and not what the policy it belongs to does: 
```if image name is image-inspector```

The reason for this is that this condition will show up in the conditions accordion as a general condition with out any relation to scanning.

Also in the policy.log you can see(currently):
```
Event: [containerimage_created], To: [openshift/image-inspector]
Resolving policy [Analyse incoming container images]
Dont scan image-inspectors image, Expression evaluation result: [true]
```